### ISSUE TITLE
Look like a scam

### ISSUE THREAD

**Issue by FoodIsGood** [madMAx43v3r/chia-gigahorse#325]
Did you notice, that fee for gpu farming is overwhelming? For me, last 50 blocs I paid 17 blocks as fee. It is not one of four blocks. It is 1.77. so, it is not 3%.it is close to 6%.what is going on?
![Screenshot_20240610-071807](https://github.com/madMAx43v3r/chia-gigahorse/assets/26028314/2fe2d468-450e-40cb-9470-f925d881aa0f)

**Comment by spleen911** [comment]
The fee % is of your rewards, not just your blocks.

Imagine if someone was really unlucky and only won a block once every 2-3 months == Max would never get paid. Sum your total rewards for the time period and calculate the % based on XCH from blocks used as a fee.

For me, 15 of 63 block rewards were used as fees, but I received 80 XCH from daily farming rewards during that time period.

**Comment by FoodIsGood** [comment]
exactly, I calculated what is the real percentege for me out of all payment. It is close to 5-6 percent. Not a 3.15

**Comment by spleen911** [comment]
Share your math with the actual numbers. Maybe you are unusually unlucky on the random block fee. My average is less than 3%. It is a random selection and an average fee. Some will be higher, other lower. GH compression will, on average, increase your daily rewards and chances of getting a block reward. We pay a DEV fee for those extra rewards and opportunity.

My farm is >200% compression and my daily rewards finally match my pre-halving rewards so even if my fee was 5% flat, I am happy to be making 0.50 XCH again. I also hit 9 block rewards in last 10 days (2 went to Max as fees).

It isn't a scam. You should edit the issue title to reflect the real issue == you question the fee calculation process. There are plenty of other issues (most closed and one of them I opened last year when I first switched to GH) of the same nature. The fee % is an average and will settle over time. My first two months were higher than 3%, but now 8 months in I am lower than 3% and my total XCH is much higher than it would be without XCH.

**Comment by madMAx43v3r** [comment]
18 out of 50 is 0.36, with 0.25 being the target. So you are paying 4.5% (`18/50/8`), still within normal randomness.

**Comment by madMAx43v3r** [comment]
> exactly, I calculated what is the real percentege for me out of all payment. It is close to 5-6 percent. Not a 3.15

You calculated wrong though, 18 out of 50 is 1 in 2.77, not 1 in 1.77. It's still more than the target, but that's just how it goes, you can be lucky or unlucky at times.
